### PR TITLE
Update documentations links

### PR DESCRIPTION
**Pull request by dijonkitchen**
Links were stale

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ title: Postgres.app – the easiest way to get started with PostgreSQL on the Ma
 
 <div class="buttons">
 	<a href="{{ site.downloadLocation }}" onclick="trackOutboundLink(this.href,'download');return false;" class="btn btn-primary btn-lg"><span class="glyphicon glyphicon-download-alt"></span> Download</a>
-	<a href="/documentation/" class="btn btn-default btn-lg"><span class="glyphicon glyphicon-book"></span> Documentation</a>
+	<a href="/docs/documentation/" class="btn btn-default btn-lg"><span class="glyphicon glyphicon-book"></span> Documentation</a>
```
Comment (jakob) [inline docs/index.md]: This change would break the link on the postgresapp.com website. The website is setup so that the /docs/ directory is at the root.

Comment (dijonkitchen) [inline docs/index.md]: Updated to absolute link in latest commit.

**Comment by jakob** [comment]
Thanks for the changes, but could you explain why you want to change this link in the first place? As far as I can see, the link on the postgresapp.com website isn't actually broken, and works fine as it was.

**Comment by dijonkitchen** [comment]
The link on the bottom of the homepage is broken (http://postgresapp.com/) and the links on GitHub were broken (https://github.com/PostgresApp/PostgresApp/blob/master/docs/index.md).

**Pull request merged**